src/utils/api.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios'
+
+const API_HOST = 'https://luxete-api-3z4w5ilef.now.sh/'
```

Comment:
What do you think about moving this `API_HOST` value to a `.env` with a simple `README.md` file?

It may be easier to change that without the need of touching the code.

Reply:
Yes, I'll move to the `.env` file.

Reply:
awesome!